### ISSUE TITLE
Plan for next SO Simulations round

### ISSUE THREAD

**Issue by zonca** [simonsobs/map_based_simulations#50]
Natural order is:

* Foreground map based sims @zonca 
* Mission scale sims (fix convergence issue) @keskitalo - end of September
* Noise map based sims @zatkins2 @AdriJD

Target a delivery by end of year of all 3 simulations, ok @aiolasimone?

### Galactic

low, medium, high complexity

### Extragalactic / CMB

CMB lensed with each Extragalactic potential

Half-Dome? if ready otherwise
if we have all 3 we could simulate all of them in order to compare them.

We need to choose 1 config for time domain, possibly medium complexity + Half Dome

**Comment by jcolinhill** [comment]
What about using Agora for the extragalactic components?  https://yomori.github.io/agora/index.html

**Comment by zonca** [comment]
Yes, the plan is to run 3 extra galactic models, Websky, Agora and Half Dome.
However we need to choose just one for time domain simulations.

**Comment by aiolasimone** [comment]
end of the year is a great.

**Comment by zonca** [comment]
@keskitalo sorry, I'm very late on this, development of the catalog took forever, and I just found another subtle issue in the catalog-background maps I need to investigate.

Our options are:

1) Wait for me to fix the outstanding issue with the catalog component of radio galaxies and run the sims, however I'm travelling 1-12 Nov, so this will take at least 3 weeks.
2) Run now without radio galaxies, I already ran the [CMB-S4 simulations](https://github.com/CMB-S4/s4mapbasedsims/tree/main/202410_s4_phase1_chile) without radio galaxies because the analysis team had a preference to not include those due to complications in the pipeline.
3) Run now with the old `rg1` model, same of last releases, which has a few sources which are too bright to be realistic, but it works

I recommend number 2 and run time domain sims on those, but @keskitalo @aiolasimone what is your preference?

**Comment by msyriac** [comment]
If you're going to run without radio, I'd argue you should leave out CIB too.  It might be nice to have a sim set with no bright objects.

Up to Reijo if he is comfortable producing another sim set later with both included again.

**Comment by keskitalo** [comment]
Running first without point sources works fine for me. If we run low on cycles, we may have to wait until January for the new allocation year but currently it is not an issue.

**Comment by zonca** [comment]
@giuspugl @keskitalo should I wait for the new bandpasses from @giuspugl ? or should I run with the bandpasses from https://github.com/simonsobs/instrument_model/ as [I did in mbs-s0012](https://github.com/simonsobs/map_based_simulations/blob/master/mbs-s0012-20230321/simonsobs_instrument_parameters_2023.03/extract_so_instrument_parameters.ipynb)

**Comment by giuspugl** [comment]
Hi all, 
I made a new repository containing  all the resampled bands (
https://github.com/simonsobs/bandpass_sampler) .  Atm only LAT bands have been sampled, but can surely produce also SAT ones if needed.

**Comment by zonca** [comment]
thanks @giuspugl, yes, please produce also the SAT.
Also I see you have a band for wafer, so I assume I'll have to take the average of the per-wafer bands to get a single frequency band, is that right?

**Comment by giuspugl** [comment]
i think the idea is to have per wafer bandpass integrated skies, not a single freq. band .

**Comment by zonca** [comment]
@giuspugl ok, thanks, I didn't know that, I can do that.

@smsimon for the beams, should I extract them from `sotodlib` as usual? https://github.com/simonsobs/sotodlib/blob/master/sotodlib/sim_hardware.py#L483-L499

**Comment by smsimon** [comment]
@zonca Yes

**Comment by zonca** [comment]
@giuspugl I can probably handle 100 or 200 points per band, so can you restrict the frequency axis and take less samples in frequency?

**Comment by giuspugl** [comment]
sure ! will resample the band !

**Comment by giuspugl** [comment]
PLease checkout the latest commit of https://github.com/simonsobs/bandpass_sampler/tree/main . I am employing 128 samples for each band. I 've also committed SAT bands

**Comment by giuspugl** [comment]
I didn't touch that yet as we might want to assess atmospheric out-of-band effects. For this reasons,  i kept the same frequency range as in the measured bands of the instrument models (even though this meant to keep this points with very low weights...)

**Comment by keskitalo** [comment]
What is the status of the signal simulations?

**Comment by zonca** [comment]
@keskitalo I am still running them, I have the last ~170 maps missing, given I'm simulating each wafer we have 72 wafers, each with a HEALPix and a CAR map:

```
output/ame_a1, 144
output/ame_a2, 144
output/cib_cib1, 144
output/cmb_c3, 144
output/cmb_c4, 144
output/co_co1, 144
output/co_co3, 144
output/dust_d10, 88
output/dust_d12, 144
output/dust_d9, 88
output/freefree_f1, 144
output/ksz_ksz1, 144
output/synchrotron_s4, 88
output/synchrotron_s5, 88
output/synchrotron_s7, 88
output/tsz_tsz1, 90
```

**Comment by keskitalo** [comment]
Excellent! Thanks for the update.

**Comment by zonca** [comment]
ok @keskitalo, I have all the maps, please double check the docs in case there is something I misunderstood, maps are at NERSC:
https://github.com/simonsobs/map_based_simulations/blob/master/mbs-s0016-20241111/README.md

I still have to run spectra validation, but it is going to take a while, so many maps, it is 16 components by 72 channels so 1152 maps.

**Comment by keskitalo** [comment]
Thanks for the update! I will be on vacation 11/20-11/25. I probably won't launch the simulations until I get back.
